File: Misc DMOJ/rgss2/src/rgss2.java
import java.util.*;
public class rgss2 {
	/* 
	 * Copy-pasting code is NOT cool! Please do not copy and paste my code as a submission to DMOJ.
	 * github.com/jdabtieu/competitive-programming
	 */
	static Scanner in = new Scanner(System.in);
	public static void main(String[] args) {
		int size = in.nextInt();
		int [] x = new int[size];
		int [] y = new int[size];
		for (int i = 0; i < size; i++) {
			x[i] = in.nextInt();
			y[i] = in.nextInt();
		}
		
		Arrays.sort(x);
		Arrays.sort(y);
		
		int length = x[size - 1] - x[0];
		int width = y[size - 1] - y[0];
		System.out.println(length * width);
	}
}

File: Misc DMOJ/rgss2/src/rgss2.cpp
#include <bits/stdc++.h>
/*
 * Copy-pasting code is NOT cool! Please do not copy and paste my code as a submission to DMOJ.
 * github.com/jdabtieu/competitive-programming
 */
using namespace std;

int main() {
    int num, minX = -1000, minY = -1000, maxX = 1000, maxY = 1000; scanf("%d", &num);

    for (int i = 0; i < num; i++) {
        int x, y; scanf("%d%d", &x, &y);

        if (x < maxX) maxX = x;
        else if (x > minX) minX = x;

        if (y < maxY) maxY = y;
        else if (y > minY) minY = y;
    }

    printf("%d", (maxX - minX) * (maxY - minY));
}
